test/dotcom_web/views/schedule/timetable_view_test.exs

```diff
@@ -64,8 +64,8 @@ defmodule DotcomWeb.Schedule.TimetableViewTest do
       origin = destination = nil
       alerts = []
 
-      all_stops = [
-        %Stops.Stop{id: "stop", name: "Stop"}
+      header_stops = [
+        {%Stops.Stop{id: "stop", name: "Stop"}, 0}
```

Comment:
This could use the stop factory

Reply:
Not yet it can't (there actually isn't a factory for creating %Stop{} yet, just the JsonApi Item structured as a stop)! :P